public/static/locales/en/common.json

```
@@ -550,6 +550,9 @@
   "delete_event_type_description": "Are you sure you want to delete this event type? Anyone who you've shared this link with will no longer be able to book using it.",
   "delete_event_type": "Delete Event Type",
   "confirm_delete_event_type": "Yes, delete event type",
+  "delete_account": "Delete account",
+  "confirm_delete_account": "Yes, delete account",
+  "delete_account_confirmation_message":"Are you sure you want to delete your Cal.com account? Anyone who you&apos;ve shared your account link with will no longer be able to book using it and any preferences you have saved will be lost.",
```

```suggestion
  "delete_account_confirmation_message":"Are you sure you want to delete your Cal.com account? Anyone who you've shared your account link with will no longer be able to book using it and any preferences you have saved will be lost.",
```
This directly renders `&apos;` in the modal itself, so I believe we are safe to use an apostrophe directly in this string.